Rename from Quality by Design to Session Insights

benhepworth opened the pull request:
#### What type of PR is this?

Add one of the following kinds:
* subproject management

#### What this PR does / why we need it:
Renaming Quality by Design to Session Insights. Supporting original docs have all been udpated to reflect the new name for historical purposes.

Link to notes from Connectivity Insights meeting where the new name was decided upon: https://lf-camaraproject.atlassian.net/wiki/x/mIDoAQ

#### Which issue(s) this PR fixes:

Fixes #116 

#### Special notes for reviewers:


#### Changelog input

```
rename Quality by Design to Session Insights

```

#### Additional documentation 

This section can be blank.



```
docs

```

hdamker commented on documentation/API proposals/APIproposal_SessionInsights_CableLabs.md:
```diff
@@ -1,16 +1,16 @@
+|                                           | **Sample Recommendations**<br>- create a HomeDeviceQoD session for WMM enablement<br>- application configuration updates<br>- request service improvements                                                                                                                                                                                                        ...
+| Technical viability                       | Application developers can use this API to interact with the network. It assumes the network operator exposing this API has a backend to receive KPIs, analyze them, and produce a Session Insights score. Based on minimum quality thresholds for the application category, the network operator can provide proactive recommendations. The application could al ...
+| Commercial viability                      | Reference implementation developed by CableLabs with a sample video conferencing app using the API is being evaluated with network operators.                                                                                                                                                                                                                     ...
+| YAML code available?                      | YES<br>- [yaml OpenAPI Specification](../SupportingDocuments/sessioninsights-openapi.yaml)<br>- [yaml - Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/cablelabs/APIBacklog/benhepworth-patch-2/documentation/SupportingDocuments/sessioninsights-openapi.yaml)<br>- [slides](../SupportingDocuments/session-insights-api.pdf)  ...
```
I propose to take the API spec files out of the API Backlog repository and create them with a PR in the repository now. That avoid having outdated YAML files in the API Backlog Supporting documents. Also to avoid the need to review them in this PR.

benhepworth replied:
will do - updating the PR shortly. thanks @hdamker !

hdamker commented on documentation/API proposals/APIproposal_SessionInsights_CableLabs.md:
```diff
@@ -1,16 +1,16 @@
-| Supporters in API Backlog Working Group   | Charter, Liberty Global, Vodafone                                                                                                                                                                                                                                                                                                                                 ...
+| **Field**                                 | **Description**                                                                                                                                                                                                                                                                                                                                                   ...
+| ----------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- ...
+| API family name                           | Session Insights - original proposal was Quality by Design (QbD)                                                                                                                                                                                                                                                                                                  ...
```
It is still the same proposal, just the name change:

```suggestion
| API family name                           | Session Insights - original proposal was named Quality by Design (QbD)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

benhepworth replied:
makes sense - updating PR

hdamker commented on documentation/API proposals/APIproposal_SessionInsights_CableLabs.md:
```diff
@@ -1,16 +1,16 @@
+|                                           | **Sample Recommendations**<br>- create a HomeDeviceQoD session for WMM enablement<br>- application configuration updates<br>- request service improvements                                                                                                                                                                                                        ...
+| Technical viability                       | Application developers can use this API to interact with the network. It assumes the network operator exposing this API has a backend to receive KPIs, analyze them, and produce a Session Insights score. Based on minimum quality thresholds for the application category, the network operator can provide proactive recommendations. The application could al ...
+| Commercial viability                      | Reference implementation developed by CableLabs with a sample video conferencing app using the API is being evaluated with network operators.                                                                                                                                                                                                                     ...
+| YAML code available?                      | YES<br>- [yaml OpenAPI Specification](../SupportingDocuments/sessioninsights-openapi.yaml)<br>- [yaml - Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/cablelabs/APIBacklog/benhepworth-patch-2/documentation/SupportingDocuments/sessioninsights-openapi.yaml)<br>- [slides](../SupportingDocuments/session-insights-api.pdf)  ...
```
The link to the PDF might better fit into another field, just a link to further information?

benhepworth replied:
makes sense - moving it into the API summary section.

benhepworth commented on documentation/SupportingDocuments/sessioninsights-openapi.yaml:
```diff
@@ -1,65 +1,54 @@
 openapi: 3.0.3
 info:
-  title: Quality by Design (QbD) API
+  title: Session Insights (SI) API
```
got it - removing the yaml and will put it in the actual repo instead.

benhepworth commented:
thanks for the feedback @hdamker - I have incorporated the changes into the PR.

hdamker commented:
@benhepworth just seen that you will end with the same (or similar?) supporting document in two different files:
* documentation/SupportingDocuments/session-insights-api.pdf
* documentation/SupportingDocuments/session-insights.pdf
Only the first one is referenced in the proposal form. You might want to take one out?

benhepworth commented:
thanks @hdamker - the second one was the proposal to the Connectivity Insights group. I have updated this PR and removed it, leaving only the original proposal to API Backlog here.

hdamker commented:
@benhepworth I have renamed the repository into https://github.com/camaraproject/SessionInsights and have done a simple replacement within https://github.com/camaraproject/SessionInsights/commit/3305b9e1578e7c1bcbf98d9e22c7ba0e46917297

Please check if any further edits in the repository are needed (https://github.com/camaraproject/SessionInsights/issues/11).

@jgarciahospital would be good to merge this PR soon.
